inji-web/src/components/DataShare/DataShareContent.tsx

```diff
@@ -58,15 +58,22 @@ export const DataShareContent:React.FC<DSContentProps> = (props) => {
                     <span className="font-base text-iw-subTitle" data-testid={"DataShareContent-Consent-Option"}>{t("content.validityTimesHeader")}</span>
                 </label>
 
-                <label className="w-1/2 flex items-center space-x-2" data-testid={"DataShareContent-Validity-Date"}>
-                    <input
-                        type="radio"
-                        name="consentValidity"
-                        value="date"
-                        disabled={true}
-                        className="accent-iw-primary scale-150"
-                    />
-                    <span className="font-light text-iw-subTitle" data-testid={"DataShareContent-Validity-Date-Title"}>{t("content.validityDate")}</span>
+                <label onClick={()=>toast.warning(t("toastText"))} className="w-1/2 flex items-center space-x-2" data-testid={"DataShareContent-Validity-Date"}>  
+                    <div className="relative inline-block">
+                        <input 
+                            type="radio" 
+                            name="consentValidity" 
+                            value="date" 
+                            disabled={true} 
+                            className="accent-iw-primary scale-150" 
+                        />
+                        <div 
+                            className="absolute inset-0"
```

Comment:
why is this div needed ?

Reply:
to make the disabled radio button react to events.  without wrapper div disabled radio button is not responding to events.